python/mqt/syrec/simulation_view/workers/simulation_run_json_export_worker.py

```
@@ -0,0 +1,187 @@
+
+                    n_skipped_sim_runs_in_batch = 0
+                    n_exported_sim_runs_in_batch = 0
+                    has_exported_first_batch = True
+                    n_remaining_sim_runs_in_batch_to_process = self._recv_queue_batch_size
+
+                # An error during during the serialization of the simulation runs to their .json representation will cause the content of the
+                # exported to .json file to be invalid .json due to the simulation runs JSON array as well as the top level JSON object missing
+                # their closing symbol.
+                file.write("]}")
+            self.finished.emit(self.is_cancellation_requested())
+        except Exception as error:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)